images/win/scripts/Installers/Run-Antivirus.ps1

```diff
@@ -0,0 +1,33 @@
+################################################################################
+##  File:  Run-Antivirus.ps1
+##  Desc:  Run a full antivirus scan.
+##         Run right after cleanup before we sysprep
+################################################################################
+
+try {
+    Update-MpSignature
+}
+catch {
+    Write-Host "Some error was found"
+    Write-Host $_
+}
+
+if ($env:run_scan_antivirus -eq $true) {
+    Write-Host "Make sure windefend is going to start"
+    Start-Service windefend -ErrorAction Continue
+    Write-Host "Waiting for windefend to report as running"
+    $service = Get-Service "Windefend"
+    $service.WaitForStatus("Running","00:10:00")
+
+    Write-Host "Run antivirus"
+    # Tell Defender to use 100% of the CPU during the scan
+    Set-MpPreference -ScanAvgCPULoadFactor 100
+    # Full Scan
+    Start-Process -FilePath "C:\Program Files\Windows Defender\MpCmdRun.exe" -ArgumentList ("-Scan","-ScanType", 2) -Wait
+}
+else {
+    Write-Host "Scanning process has been skipped"
+}
+
+Write-Host "Set antivirus parameters"
+Set-MpPreference -ScanAvgCPULoadFactor 5 -ExclusionPath "D:\", "C:\"
```

Comment:
Ideally this would be in its own script/call after the RunAntivirus in the json file. If I'm reading the file I would be surprised to see that this changed even though the run_antivirus variable was set to false.